config-domain.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# script to run the sed commands in both OSX and linux (e.g., GitHub Action)
+# the commands ...
+# first remove the :8000 part of the http://127.0.0.1:8000 domain
+# then replace all http://127.0.0.1 with the domain passed to the script
+
+echo $1
+
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    sed -i '' "s/:8000//g"  src/manifest.json src/js/background.js src/popup.html
+    sed -i '' "s|http://127.0.0.1|$1|g" src/manifest.json src/js/background.js src/popup.html
+else
+    sed -i "s/:8000//g"  src/manifest.json src/js/background.js src/popup.html
+    sed -i "s|http://127.0.0.1|$1|g" src/manifest.json src/js/background.js src/popup.html
+fi
```

Comment:
I found one edge case this can't account for: 

- Run `config:stage`
- Run `config:prod`
- **Expected:** The manifest.json / background.js / popup.html should all have the URL param set to `https://relay.firefox.com`
- **Actual:** The param is still set to stage's value (`https://stage.fxprivaterelay.nonprod.cloudops.mozgcp.net`)

Reply:
Fixed in https://github.com/mozilla/fx-private-relay-add-on/pull/120/commits/aedff362b40afbf1e573d586353d16eeb45918f6.